cmd/optional/main.go

```diff
@@ -146,11 +146,28 @@ func New{{ .OutputName }}(v {{ .TypeName }}) {{ .OutputName }} {
 	return {{ .OutputName }}{&v}
 }
 
+// New{{ .OutputName }}FromPointer creates an optional.{{ .OutputName }} from a {{ .TypeName }} pointer.
```

Comment:
Better late than never ;)

WDYT about `FromPtr` / `ToPtr` ? saves some keystrokes and I think is just as readable?

Reply:
I like it. I’ll push up and update

Reply:
Updated 👍